Header header
string object_id
---
Header header
bool success